openreview/conference/matching.py

```diff
@@ -890,26 +932,52 @@ def setup(self, affinity_score_file=None, tpms_score_file=None, elmo_score_file=
             invitation=self.conference.get_blind_submission_id(),
             details='original'))
 
+        if not self.match_group.members:
+            raise openreview.OpenReviewException(f'The match group is empty: {self.match_group.id}')
+        if self.alternate_matching_group:
+            other_matching_group = self.client.get_group(self.alternate_matching_group)
+            if not other_matching_group.members:
+                raise openreview.OpenReviewException(f'The alternate match group is empty: {self.alternate_matching_group}')
+        elif not submissions:
```

Comment:
I don't think we can remove this elif, because we only want to check if there are submissions if there is no `alternate_matching_group`.

Reply:
You are right! ignore my comment.